src/main/scala/regmapper/Annotation.scala

```diff
@@ -2,17 +2,121 @@
 
 package freechips.rocketchip.regmapper
 
+import chisel3.internal.InstanceId
+import chisel3.experimental.{ChiselAnnotation, RunFirrtlTransform, annotate}
+import firrtl.{CircuitForm, CircuitState, LowForm, Transform}
+import firrtl.annotations._
+import freechips.rocketchip.regmapper.RegFieldAccessType.RegFieldAccessType
+import freechips.rocketchip.regmapper.RegFieldRdAction.RegFieldRdAction
+import freechips.rocketchip.regmapper.RegFieldWrType.RegFieldWrType
+import org.json4s.JsonAST.JValue
 import org.json4s.JsonDSL._
 import org.json4s.jackson.JsonMethods.{pretty, render}
 
+// Do not call this function
+// This function is called by DescribedRegChiselAnnotation
```

Comment:
then can't it be placed inside that class and marked `private`?

Reply:
Ask Jack

Reply:
Comment updated